.github/DEVELOPMENT.md

```diff
@@ -0,0 +1,131 @@
+# Development
+
+Developers should read [the development section of the website](https://trino.io/development),
+which covers things like development philosophy and contribution process.
+
+* [Commits and pull requests](#commits-and-pull-requests)
+* [Code style](#code-style)
+
+## Getting started
+
+Start by forking the repository and then modify the code in your fork.
+
+We recommend that you use Python3's `venv` for development:
+
+```
+$ python3 -m venv .venv
+$ . .venv/bin/activate
+$ pip install -e '.[tests]'
+```
+
+With `-e` passed to `pip install` above pip can reference the code you are
+modifying in the *virtual env*. That way, you do not need to run `pip install`
+again to make your changes applied to the *virtual env*.
+
+## Running tests
+
+`trino-python-client` uses [pytest](https://pytest.org/) for its tests. To run
+only unit tests, type:
+
+```
+$ pytest tests/unit
+```
+
+Then you can pass options like `--pdb` or anything supported by `pytest --help`.
+
+To run integration tests:
+
+```
+$ pytest tests/integration
+```
+
+They pull a Docker image and then run a container with a Trino server:
+- the image is named `trinodb/trino:${TRINO_VERSION}`
+- the container is named `trino-python-client-tests-{uuid4()[:7]}`
+
+To run the tests with different versions of Python in managed *virtual envs*,
+use `tox` (see the configuration in `tox.ini`):
+
+```
+$ tox
+```
+
+When the code is ready, submit a Pull Request.
+
+## Commits and pull requests
+
+See [Commits and pull requests](https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#commits-and-pull-requests) section from Trino.
+
+## Code style
+
+To run linting and formatting checks before opening a PR: `#TODO - fill this in`
```

Comment:
@hovaesco Please help fill this in - I think we don't require anything at all yet?

Reply:
Not yet, open PR: https://github.com/trinodb/trino-python-client/pull/227

Reply:
Not exactly true - need to do `pip install pre-commit && pre-commit run --all-files` (this is nice since it doesn't configure pre-commit to run prior to commit but can still be invoked manually when needed).